include/RAJA/policy/openmp_target/reduce.hpp

```diff
@@ -196,7 +194,6 @@ struct TargetReduce
   //! apply reduction on device upon destruction
   ~TargetReduce()
   {
-    assert ( omp_get_num_teams() <= omp::MaxNumTeams );
```

Comment:
Why is this assertion removed? Does it still need to be enforced?

Reply:
We don't really have a policy on asserts, but that sounds like a good idea. If you can comment them out and leave a note about the XL issue that would be perfect.

Reply:
It's not really necessary at that point because we have performed checks on the number of teams in include/RAJA/policy/openmp_target/forall.hpp lines 54-69.

If you add the asserts back in, you'll see that clang coral cannot nvlink those asserts. The asserts used to work with XL, however, after ~mid-January XL builds started to hang during test-atomic. I can put the asserts back in as comments, until compiler issues are fixed in the future?